### PR TITLE
chore: inactive day change to 3

### PR DESCRIPTION
**Pull request by afc163**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     -      |
| 🇨🇳 Chinese |     -      |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: .github/workflows/issue-reply.yml [hunk 1]
```diff
@@ -30,9 +30,9 @@ jobs:
           token: ${{ secrets.ANT_BOT_TOKEN }}
           issue-number: ${{ github.event.issue.number }}
           body: |
-            Hello @${{ github.event.issue.user.login }}. Please provide a online reproduction by forking this link https://u.ant.design/codesandbox-repro or a minimal GitHub repository. Issues labeled by \`Need Reproduce\` will be closed if no activities in 7 days.
+            Hello @${{ github.event.issue.user.login }}. Please provide a online reproduction by forking this link https://u.ant.design/codesandbox-repro or a minimal GitHub repository. Issues labeled by `Need Reproduce` will be closed if no activities in 3 days.
```
Comment (afc163) [inline .github/workflows/issue-reply.yml:33]: 这里多了 `

File: .github/workflows/issue-reply.yml [hunk 2]
```diff
@@ -43,6 +43,7 @@ jobs:
           actions: 'create-comment,close-issue'
           token: ${{ secrets.ANT_BOT_TOKEN }}
           issue-number: ${{ github.event.issue.number }}
+          inactive-day: 3
```
Comment (xrkffgg) [inline .github/workflows/issue-reply.yml:46]: 这个地方加这个是想

在增加这2个label 的时候，如果这 issue 3 天未活跃就关闭，如果是在 3 天内，这个 issue 只评论不关闭？

Comment (afc163) [reply .github/workflows/issue-reply.yml:46]: 不知道应该加在哪，本意是 7 天改 3 天。

Comment (xrkffgg) [reply .github/workflows/issue-reply.yml:46]:
```suggestion

```

**Pull request merged**